.changeset/shy-needles-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': major
```

Comment:
breaking change because the rafiki `POST /outgoing-payment` validation will now fail because it is expecting `grantSpentDebitAmount` and `grantSpentReceiveAmount` fields which are not returned.